### PR TITLE
[Rule Tuning] Update Cloud Rule Syntax

### LINKED ISSUE #982: [Rule Tuning] Check cloud rule syntax for refinement opportunities
## Description
Rules that have operation names or event actions with multiple backslashes or periods like Azure, GCP, Workspace, etc. could be refined with quotes as to not risk improper firing of the rules.  This is a placeholder issue to go through all the cloud rules and adjust accordingly. 

## Example Data
`google.iam.admin.v*.DisableServiceAccount`
`MICROSOFT.INSIGHTS/DIAGNOSTICSETTINGS/DELETE`

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #982 

## Summary
Rules that have operation names or event actions with multiple backslashes or periods like Azure, GCP, Workspace, etc. could be refined with quotes as to not risk improper firing of the rules. 

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/azure/persistence_azure_automation_webhook_created.toml [hunk 1]
```diff
@@ -27,7 +27,6 @@ rule_id = "e9ff9c1c-fe36-4d0d-b3fd-9e0bf4853a62"
 severity = "low"
 tags = ["Elastic", "Cloud", "Azure", "Continuous Monitoring", "SecOps", "Configuration Audit"]
 timestamp_override = "event.ingested"
-to = "now-25m"
```
Comment (threat-punter) [inline rules/azure/persistence_azure_automation_webhook_created.toml:30]: What's the reason for this change? 🤔

**Review by threat-punter — APPROVED** [review]
LGTM. I left one comment for your review.

**Review by brokensound77 — APPROVED** [review]
Thanks for the cleanup 👍

File: rules/azure/persistence_azure_automation_webhook_created.toml (hunk 1 above)
Comment (bm11100) [reply rules/azure/persistence_azure_automation_webhook_created.toml:30]: That appears to be a mistake, it already has `from = "now-25m"` with the default run interval. Accidental addition. 

image
